src/change-listeners.js

```diff
@@ -269,6 +283,9 @@ class ChangeListeners extends Process {
     try {
       await this._processBatchOfChanges(listener)
     } catch (err) {
+      if(err instanceof DatabaseNotFoundError) {
```

Comment:
Is there a reason that we need this if check here?

Reply:
Without it, the DatabaseNotFoundError is swallowed by the rest of the catch handler, rather than being handled by the caller.  I suppose I could rethrow all errors, but that seemed an unnecessary breaking change.